libs/build_api_v2.py

```diff
@@ -35,17 +35,28 @@
 USA_VACCINATION_START_DATE = datetime(2020, 12, 14)
 
 
-def _build_actuals(actual_data: dict) -> Actuals:
+def _select_category(category: str, data: dict):
```

Comment:
Add comment referring to `make_short_name` which generated the string this method parses. Or even better, move them to the same module.